.gitignore

```diff
@@ -1 +1,2 @@
 .idea/*
+.venv
```

Comment:
It's not related, so either remove this change, or move it to a separate commit.